hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/io/ECKeyOutputStream.java

```diff
@@ -289,8 +289,10 @@ private void excludePipelineAndFailedDN(Pipeline pipeline,
     failedStreams.stream()
         .filter(s -> !checkIfContainerToExclude(
             HddsClientUtils.checkForException(s.getIoException())))
-        .forEach(s -> blockOutputStreamEntryPool.getExcludeList()
-            .addDatanode(s.getDatanodeDetails()));
+        .forEach(s -> {
+            LOG.info("Node added to exclude list: {}", s.getDatanodeDetails(), s.getIoException());
+            blockOutputStreamEntryPool.getExcludeList().addDatanode(s.getDatanodeDetails());
+          });
```

Comment:
Thanks @MikaelSmith for the patch.

This formatting triggers a bunch of [checkstyle violations](https://github.com/apache/ozone/actions/runs/3989592985/jobs/6842563103#step:7:11):

```
hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/io/ECKeyOutputStream.java
 293: Line is longer than 80 characters (found 99).
 293: 'block' child has incorrect indentation level [12](https://github.com/apache/ozone/actions/runs/3989592985/jobs/6842563103#step:7:13), expected level should be 10.
 294: Line is longer than 80 characters (found 92).
 294: 'block' child has incorrect indentation level 12, expected level should be 10.
 295: 'block rcurly' has incorrect indentation level 10, expected level should be 8.
```

I think we can get of all of those and keep the code more readable by extracting these two lines into a private method.